vue3-apollo-quasar/tests/unit/components/PaginationButtons.spec.ts

```diff
@@ -1,33 +1,11 @@
 import { mount } from '@vue/test-utils';
 import { PaginationButtons } from '@/components';
 
```

Comment:
Please add tests that test for the actual functionality:
- Displays pagination buttons
- Tests that clicking pagination works
- Tests that next and previous buttons work


Testing for mount functionality only doesn't provide any value

Reply:
Can't do deep testing.